src/main/java/com/exasol/bucketfs/WriteEnabledBucket.java

```diff
@@ -47,8 +47,7 @@ public void uploadFileNonBlocking(final Path localPath, final String pathInBucke
     protected void uploadContentNonBlocking(final BodyPublisher bodyPublisher, final String pathInBucket,
             final String contentDescription) throws InterruptedException, BucketAccessException {
         final URI uri = createWriteUri(pathInBucket);
-        LOGGER.info(() -> "Uploading \"" + contentDescription + "\" to bucket \"" + this.bucketFsName + "/"
-                + this.bucketName + "\": \"" + uri + "\"");
+        LOGGER.info(() -> "Uploading \"" + contentDescription + "\" to bucket \"" + this + "\" at \"" + uri + "\"");
```

Comment:
How is it supposed to work with `this`? I think it will return the Object name, won't it?

Reply:
No. Thanks to the `toString` method inherited from the `ReadEnabledBucket` this will display the same kind of info.